library/ecp_curves.c

```diff
@@ -5594,6 +5596,12 @@ static inline int ecp_mod_koblitz(mbedtls_mpi *N, mbedtls_mpi_uint *Rp, size_t p
  * with R = 2^32 + 2^12 + 2^8 + 2^7 + 2^6 + 2^3 + 1 = 0x0100001119
  */
 static int ecp_mod_p192k1(mbedtls_mpi *N)
+{
+    return mbedtls_ecp_mod_p192k1(N);
+}
+
+MBEDTLS_STATIC_TESTABLE
+int mbedtls_ecp_mod_p192k1(mbedtls_mpi *N)
```

Comment:
An alternative way would be to rename `ecp_mod_p192k1` -> `mbedtls_ecp_mod_p192k1` and guard it by `MBEDTLS_STATIC_TESTABLE`

But they are both functionally equivalent and those methods will soon be removed anyway, so this should not be a blocker

Reply:
I think it is easier to introduce a new function and when updating to `_raw` function only a rename must be done.